src/main/java/services/AlgorithmPicker.java

```diff
@@ -59,7 +59,7 @@ public AlgorithmPicker(){
     matchingAlgorithms.add(new CompoundMatcherSigmoid());
     matchingAlgorithms.add(new ContextSubsumptionMatcherSigmoid());
     matchingAlgorithms.add(new DefinitionSubsumptionMatcherSigmoid());
-    matchingAlgorithms.add(new DefinitionSubsumptionMatcherSigmoid_revised());
+    matchingAlgorithms.add(new DefinitionSubsumptionMatcherSigmoid());
```

Comment:
This line can just be removed (same with line 19). It's a duplicate of 61 (and 18).